apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/tag/Tags.java

```diff
@@ -96,6 +96,8 @@ private Tags() {
      */
     public static final StringTag RPC_RESPONSE_STATUS_CODE = new StringTag(18, "rpc.status_code", true);
 
+    public static final StringTag SIGNAL_TYPE = new StringTag(19, "signalType");
```

Comment:
What is this tag? It seems don't have general meaning.

Reply:
removed